src/view/components/toolbar/motion/MotionSensorBar.tsx

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as React from "react";
+import { CONSTANTS } from "../../../constants";
+import "../../../styles/MotionSensorBar.css";
+import { ISensorProps, ISliderProps } from "../../../viewUtils";
+import svg from "../../cpx/Svg_utils";
+import SensorButton from "../SensorButton";
+import { ThreeDimensionSlider } from "./threeDimensionSlider/ThreeDimensionSlider";
+
+interface vscode {
+    postMessage(message: any): void;
+}
+
+declare const vscode: vscode;
+
+const sendMessage = (state: any) => {
```

Comment:
Didn't you abstract this out already?

Reply:
I've removed all duplicates declarations from the utils. Resolved forever :)